dataverse-webapp/src/main/webapp/selectGuestbook.xhtml

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://java.sun.com/jsf/html"
+      xmlns:f="http://java.sun.com/jsf/core"
+      xmlns:ui="http://java.sun.com/jsf/facelets"
+      xmlns:p="http://primefaces.org/ui"
+      xmlns:c="http://xmlns.jcp.org/jsp/jstl/core"
+      xmlns:jsf="http://xmlns.jcp.org/jsf"
+      xmlns:o="http://omnifaces.org/ui"
+      xmlns:dv="http://dataverse.org/facelets"
+>
+<f:view locale="#{dataverseSession.localeCode}"/>
+
+<!--@elvariable id="dataset" type="edu.harvard.iq.dataverse.Dataset"-->
+
+<h:head>
+</h:head>
+<h:body>
+    <ui:composition template="/dataverse_template.xhtml">
+        <ui:param name="pageTitle" value="#{selectGuestbookPage.workingVersion.title} - #{selectGuestbookPage.dataset.owner.name}"/>
+        <ui:param name="dataset" value="#{selectGuestbookPage.dataset}"/>
+        <ui:param name="showMessagePanel" value="#{true}"/>
+        <ui:define name="body">
+            <f:metadata>
+                <f:viewParam name="datasetId" value="#{selectGuestbookPage.datasetId}"/>
+                <f:viewParam name="persistentId" value="#{selectGuestbookPage.persistentId}"/>
+                <f:viewAction action="#{selectGuestbookPage.init}" rendered="true"/>
+                <f:viewAction action="#{dataverseHeaderFragment.initBreadcrumbs(selectGuestbookPage.dataset)}"/>
+            </f:metadata>
+            <h:form id="selectGuestbookForm">
+                <!-- Edit editMode -->
+                <div class="button-block form-top">
+                    <p:commandButton id="saveTop" value="#{bundle.saveChanges}"
+                                     onclick="checkNewlyRestricted();PF('lockSelectGuestbookForm').show();"  />
+                    <p:commandButton id="cancelTop" value="#{bundle.cancel}"
+                                     action="#{selectGuestbookPage.cancel}" />
+                </div>
+                <!-- END Header / Button Panel -->
+
+                <!-- Tabs -->
+                <div class="text-right margin-bottom"
+                     jsf:rendered="#{dataverseSession.user.authenticated and !widgetWrapper.widgetView
+                        and permissionsWrapper.canIssueUpdateDatasetCommand(dataset)}">
+                </div>
```

Comment:
Not sure if this does anything. And if it does - then there is no need to have any of this conditions in rendered. You already checking permissions in init and widgetView is for dataset page only